routes/repositories.json

```diff
@@ -0,0 +1,278 @@
+[
+  {
+    "name": "List your repositories",
+    "method": "GET",
+    "path": "/user/repos",
+    "description": "List repositories that are accessible to the authenticated user.\n\nThis includes repositories owned by the authenticated user, repositories where the authenticated user is a collaborator, and repositories that the authenticated user has access to through an organization membership.",
+    "params": [
+      {
+        "name": "visibility",
+        "type": "string",
+        "description": "Can be one of `all`, `public`, or `private`.",
```

Comment:
Changing enumerated values to be `"type": [ "all", "public", "private" ]` rather than `"string"` would help in generating the TypeScript definition file. 

I will try to update the `index.js` script but you might beat me to it 😄